GOVERNANCE.md

```diff
@@ -0,0 +1,356 @@
+# Main Governance Document
+
+## The Project
+
+The Bambi Project (The Project) is an open source software project.
+The goal of The Project is to develop open source software and deploy open and public websites and 
```

Comment:
```suggestion
The goal of The Project is to develop open source software and deploy open and public content and 
```
No change needed. websites seems specific, perhaps this can be broadened